.coveragerc

```diff
@@ -9,6 +9,10 @@ concurrency =
 
 [report]
 ignore_errors = True
+omit = 
+    tests/*
+    scripts/*
```

Comment:
Should we later include scripts again as part of the coverage?

Reply:
I'm not sure. What's your opinion? Should we just include those that will be run as our scheduled test?

Reply:
Perhaps we can have a separate coverage report for the scripts. This will simplify the report generation as scripts testing schedule would differ from the general tests.